back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -8,9 +8,11 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     TextImageService.new.render_data_images_multiloc field.description_multiloc, field: :description_multiloc, imageable: field
   end
 
-  attribute :answer_visible_to, if: proc { |_object, params|
+  attribute :answer_visible_to, if: proc { |_field, params|
```

Comment:
@IvaKop It looks like the frontend is not using this attribute today (it only gets this through the UI schema), so we could replace it by a simple boolean instead of "public" or "admin" if you think that's cleaner?

Reply:
Agreed 👍